lib/stylesheet/manager/builder.rb

```diff
@@ -225,7 +225,7 @@ def uploads_digest
       sha1s << upload_field.upload&.sha1
     end
 
-    Digest::SHA1.hexdigest(sha1s.sort!.join("\n"))
+    Digest::SHA1.hexdigest(sha1s.compact.sort!.join("\n"))
```

Comment:
Nitpick, but I'd remove the `.compact` from here and instead change the `.map` at line 224 to `.filter_map` which removes nil values from the mapped array.

Reply:
It won't work in this case. Here the nil values are not found while doing the `.map`. It is only happening after we collected the sha1 strings since the upload record is deleted.

Reply:
oops my bad, I thought sha1s was the result of the map, but we're actually iterating over the array and appending to sha1s. (although that means `.map` can/should be replaced with a simple `.each`?)

Reply:
Yes. I'm not sure why they used `.map` in the code initially. I will update it when I refactor that code next time.